### PR TITLE
Add missing tooltips, adjust statuses

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- add tooltips to the resources
- adjust statuses of deployments, replica sets, stateful sets, daemon sets
- simplify some statuses

### CODE REVIEW

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -77,9 +79,10 @@ api-rules:
     loading: Loading services...
     ns-no-services: No Services in this Namespace.
     placeholder: Choose the service to expose.
-  statuses:
-    skipped: Skipped
   title: API Rules
+  tooltips:
+    ok: Resource created.
+    skipped: Skipped creating a resource.
```
Comment (majakurcius) [inline core/src/i18n/en.yaml:85]:
```suggestion
    skipped: Skipped creating the resource.
```

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -823,7 +837,11 @@ functions:
     unhealthy: Unhealthy
   title: Functions
   tooltips:
+    building: Waiting for the image to be build.
```
Comment (majakurcius) [inline core/src/i18n/en.yaml:840]:
```suggestion
    building: Waiting for the image to be built.
```

But perhaps I'd shorten it to `Waiting for the image to build`.

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -823,7 +837,11 @@ functions:
...
+    deploying: Deploying function.
```
Comment (majakurcius) [inline core/src/i18n/en.yaml:841]:
```suggestion
    deploying: Deploying the Function.
```
We [capitalize `Function`](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalized-terms) as a proper name.

**Review by majakurcius — DISMISSED** [review]

**Review by majakurcius — DISMISSED** [review]

File: shared/components/TooltipBadge/TooltipBadge.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+  const badgeElement = (
+    <ObjectStatus
+      ariaLabel="Status"
+      role="status"
+      inverted
+      status={type}
+      className={classes}
+      data-testid={'has-tooltip'}
+      style={{ whiteSpace: 'nowrap' }}
```
Comment (qbalukom) [inline shared/components/TooltipBadge/TooltipBadge.js:27]: why this mix of css and `style=`? this should be moved to css file as well.

**Review by qbalukom — CHANGES_REQUESTED** [review]

**Review by qbalukom — APPROVED** [review]

**Review by majakurcius — DISMISSED** [review]

**Review by majakurcius — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Pull request merged**